newIDE/app/src/Utils/GDevelopServices/Game.js

```diff
@@ -25,6 +26,10 @@ export type PublicGame = {
   categories?: string[],
 };
 
+export type ApiOnlyPublicGameInfo = {
```

Comment:
Can you use a strict type `{| ... |}`?

Reply:
Also, one thing about this type: the name "ApiOnly" looks like something that would be just sent by the API, or just received by the API. But this is not exposed at any point by the API, it's not an entity that exists (and that's a good thing :)), and it's only used for a component basically.

So could we:
- move this into the component file using it in its props (and import in the parent file)
- find a name that explains what's in without talking about the api itself?

Reply:
Basically, this can be understood as a "PublicGameChanges" or "GameChanges" or maybe better: "PartialGameChanges". I.e:
It's a description of the changes to do to a Game, though it's only a partial description (the rest of the changes are "casted" from the Project to the Game).

And we can keep it in a component, because the API has no knowledge of the existence of this (it's just a detail of the IDE, but the API itself expect you to send a `Game` to its update endpoint :) And this is good!)